additionalPosts-biblio.php

```diff
@@ -1,8 +1,13 @@
 <?php
-    /*
-        Template Name: Additional Posts Bibliotech
```

Comment:
The following is being deleted because these shouldn't be page templates,
as such, allowed to be chosen in the dropdown by the user in the WP
Dashboard, they are more like template-parts that are improperly declared
to be actual page templates...

Template Name: Additional Posts Bibliotech


2016-07-15 16:20 GMT-04:00 Matthew Bernhardt <notifications@github.com>:

> In additionalPosts-biblio.php
> <https://github.com/MITLibraries/MITLibraries-news/pull/69#discussion_r71035123>
> :
>
> > @@ -1,8 +1,13 @@
> >  <?php
> > -    /*
> > -        Template Name: Additional Posts Bibliotech
>
> I'm assuming that where these Template Name: **** lines are being
> deleted, it is because the files in question don't need that metadata? Page
> templates do, I know - but I don't think these do?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MITLibraries/MITLibraries-news/pull/69/files/681ea9151eb773f0fde949e71eccd585b468c1e9#r71035123>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABv1VfJ9SkzORrnzPaAmUKP5zDR9RScMks5qV-t1gaJpZM4JLyGJ>
> .
>



-- 

"The reason is never the reason."


*- Some French Movie Director*"Every portrait that is painted with feeling
is a portrait of the artist, not the sitter."
*- Oscar Wilde*